.github/workflows/ssm-command.yaml

```diff
@@ -0,0 +1,452 @@
+name: SSH and SSM Command Execution
+
+on:
+  workflow_dispatch:
+    inputs:
+      command:
+        description: 'Command to execute on agent instances'
+        required: true
+        type: string
+      instance_name:
```

Comment:
Remove this. We can infer the instance name by inferring the pr number from the target branch